include/broker/detail/radix_tree.hh

```diff
@@ -699,8 +699,8 @@ radix_tree<T, N>::find_child(node* n, unsigned char c) {
 }
 
 template <typename T, std::size_t N>
-size_t radix_tree<T, N>::prefix_shared(node* n, const key_type& key,
-                                       int depth) {
+size_t
+radix_tree<T, N>::prefix_shared(node* n, const key_type& key, int depth) {
```

Comment:
I'm not sure how I feel about this one. I think I would prefer the old way here, breaking the line in the argument list instead of after the return type.